emacs/ocamlformat.el

```diff
@@ -261,14 +270,11 @@ function."
                (if ocamlformat--support-replace-buffer-contents
                    (ocamlformat--replace-buffer-contents outputfile)
 		 (ocamlformat--patch-buffer outputfile))
+               (ocamlformat--process-errors
+                 (buffer-file-name) bufferfile errorfile errbuf)
                (message "Applied ocamlformat"))
-             (if errbuf
-               (progn
-                 (with-current-buffer errbuf
-                   (setq buffer-read-only nil)
-                   (erase-buffer))
-                 (ocamlformat--process-errors
-                   (buffer-file-name) bufferfile errorfile errbuf)))
+             (ocamlformat--process-errors
+               (buffer-file-name) bufferfile errorfile errbuf)
```

Comment:
You could also call it from outside this if, after `message`.

Reply:
done